test/provision/compile.sh

```diff
@@ -16,7 +16,7 @@ cd ${GOPATH}/src/github.com/cilium/cilium
 if echo $(hostname) | grep "k8s" -q;
 then
     if [[ "$(hostname)" == "k8s1" ]]; then
-        make docker-image
+        make LOCKDEBUG=1 docker-image
```

Comment:
I assume this is equivalent to `make docker-image LOCKDEBUG=1`.

Reply:
Correct, I verified that the builds on Jenkins are compiled with lockdebug enabled and when running `make docker-image` locally that the image is not built with lockdebug.